backend/.rubocop.yml

```
@@ -1,54 +1,1169 @@
+  EnforcedStyle: explicit
+
+Lint/RedundantRequireStatement:
+  EnforcedStyle: explicit
+
+Lint/RedundantSplatExpansion:
+  EnforcedStyle: explicit
+
+Lint/RedundantStringCoercion:
+  EnforcedStyle: explicit
+
+Lint/RedundantWithObject:
```

YAMLlint (1.37.1)

[error] duplication of key "Lint/RedundantWithObject" in mapping

(key-duplicates)